diesel/src/connection/statement_cache.rs

```diff
@@ -1,3 +1,95 @@
+//! A primer on prepared statement caching in Diesel
+//! ------------------------------------------------
```

Comment:
Well, with my 4 year out of date HTML5 knowledge I'd expect rustdoc to put that in an `<article>` :D